src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/visit/BookVisitValidationTest.kt

```diff
@@ -123,11 +123,11 @@ class BookVisitValidationTest : IntegrationTestBase() {
     val responseSpec = callVisitBook(webTestClient, roleVisitSchedulerHttpHeaders, application.reference)
 
     // Then
-    responseSpec.expectStatus().isBadRequest
+    responseSpec.expectStatus().isEqualTo(422)
```

Comment:
Small suggestion which you can ignore if you want. Instead of

isEqualTo(422)

isEqualTo(UNPROCESSABLE_ENTITY.value()) to make it clear what the 422 is at a glance.

Approving anyway as its a preference.

Reply:
Updated